docs/source/conf.py

```diff
@@ -43,20 +43,21 @@
     # "sphinx_autodoc_typehints",
 ]
 
+# -- Options for autodoc --------------------------------------------------
+autosummary_generate = True
+autoclass_content="class"
 autodoc_default_options = {
     "members": True,  # Include all methods.
     "member-order": "alphabetical",  # 'alphabetical', by member type ('groupwise') or source order (value 'bysource')
     "undoc-members": True,  # If set, autodoc will also generate document for the members not having docstrings
-    "private-members": False,  # don't generate document for the private members (like _private or __private)
-    # 'special-members': '',  # for example, '__init__'
-    "inherited-members": False,
+    # "private-members": '',  # don't generate document for the private members (like _private or __private)
+    # "special-members": '',  # for example, '__init__'
+    # "inherited-members": '',
```

Comment:
I'm a bit confused about the changes here. Why do we comment certain settings?

Reply:
This is because specifying an empty string or `None` is same as not specifying. This is a good point, I should probably just remove the comments to avoid confusion.